### DISCUSSION TITLE
No longer earning pp after 2023.121.0 Update

### DISCUSSION THREAD

**Discussion by falkon13** [ppy/osu#22398] [Q&A] [answered]
Ever since I've installed Lazer 2023.121.0 and then 2023.123.0 I've stopped earning pp for any plays I've done. My scores sync over to Lazer and the web client and appear in my "Best Performances" and any beatmap leaderboard; but my pp score hasn't updated.

For catch my pp is 827 when I've roughly calculated it should be ~870
https://lazer.ppy.sh/users/1831550/fruits

Has there been a freeze on pp updates recently? I didn't see anything in the changelogs but maybe I missed it

**Comment by bdach** [comment, marked as answer]
Let's move this to https://github.com/ppy/osu/discussions/22352 as it seems to be about the same thing.
